resources/baseJenkins.yaml

```diff
@@ -175,8 +175,8 @@ tool:
       properties:
       - installSource:
           installers:
-          - adoptOpenJdkInstaller:
-              id: "jdk-14.0.2+12"
+          - zip:
+              url: "https://ci.opensearch.org/ci/dbc/tools/OpenJDK14U-jdk_x64_linux_hotspot_14.0.2_12.zip"
     - name: "openjdk-17"
```

Comment:
Hey @peterzhuamazon do we need to add a sub-directory `Subdirectory of extracted archive: jdk-14.0.2+12` option?

Reply:
nope it will auto unzip to this:
```
/var/jenkins/tools/hudson.model.JDK# ll openjdk-14/
total 40
drwxrwxr-x  9 ubuntu ubuntu 4096 Sep 12 17:40 ./
drwxrwxr-x  6 ubuntu ubuntu 4096 Sep 12 17:40 ../
-rwxrwxr-x  1 ubuntu ubuntu    0 Sep 12 17:39 .timestamp*
drwxrwxr-x  2 ubuntu ubuntu 4096 Sep 12 17:40 bin/
drwxrwxr-x  5 ubuntu ubuntu 4096 Sep 12 17:40 conf/
drwxrwxr-x  3 ubuntu ubuntu 4096 Sep 12 17:40 include/
drwxrwxr-x  2 ubuntu ubuntu 4096 Sep 12 17:40 jmods/
drwxrwxr-x 74 ubuntu ubuntu 4096 Sep 12 17:40 legal/
drwxrwxr-x  5 ubuntu ubuntu 4096 Sep 12 17:40 lib/
drwxrwxr-x  3 ubuntu ubuntu 4096 Sep 12 17:40 man/
-rwxr-xr-x  1 ubuntu ubuntu 1272 Jul 15  2020 release*
```